zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json

```diff
@@ -1,16 +1,839 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
```

Comment:
If they do, this unit test should catch it: https://github.com/zenodo/zenodo/pull/569/files#diff-a38230d8a36856aa1ee35822e1d6c9d3R32